packages/zod-adapter/src/index.ts

```diff
@@ -1,4 +1,5 @@
-import { z } from 'zod'
+import * as z3 from 'zod/v3'
+import * as z4 from 'zod/v4'
```

Comment:
This import should be modified slightly to include the `/core` path.

`import * as z4 from "zod/v4/core";`